src/context/authContext.js

```diff
@@ -0,0 +1,55 @@
+import { createContext, useContext, useState } from "react";
+import { loginUtility } from "../utils/authHelper";
+import { useNavigate } from "react-router-dom";
+
+const AuthContext = createContext();
+
+const AuthProvider = ({ children }) => {
+  const navigate = useNavigate();
+
+  const localStorageToken = JSON.parse(localStorage.getItem("loginItems"));
+  const [token, setToken] = useState(localStorageToken?.token);
+  const [currUser, setCurrUser] = useState(localStorageToken?.user);
```

Comment:
currUser is not that readable, I suggest using a more readable name here. `currentUser`